distribution/docker/build.gradle

```diff
@@ -296,28 +304,25 @@ void addTransformDockerContextTask(Architecture architecture, DockerBase base) {
   def transformTask = tasks.register(taskName("transform", architecture, base, "DockerContext"), Sync) {
     TaskProvider<Tar> buildContextTask = tasks.named(taskName("build", architecture, base, "DockerContext"))
     dependsOn(buildContextTask)
-
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
```

Comment:
Does the exploded distro include duplicate entires? Why is this needed?

Reply:
This is needed as we have tweaked config files like `config/elasticsearch.yml` and `config/log4j2.properties` and we want to keep them when copying over the elasticsearch distro alternatively we could exclude it from that specific copy definition but that seemed simpler and should also be faster by not checking excludes with every file.

Reply:
Got it, that just means that order matters here so perhaps we should add some comments to that effect so we don't accidentally mess this up later.